docs/cloud/alerts-notifications/add-webhook-notification-configuration.md

```diff
@@ -78,10 +78,91 @@ Netdata webhook integration supports 3 different authorization mechanisms:
 
 In mutual Transport Layer Security (mTLS) authorization, the client and the server authenticate each other using X.509 certificates. This ensures that the client is connecting to the intended server, and that the server is only accepting connections from authorized clients.
 
-To take advantage of mutual TLS, you can configure your server to verify Netdata's client certificate. To do that you need to download our [CA certificate file](http://localhost) and configure your server to use it as the
-
 This is the default authentication mechanism used if no other method is selected.
 
+To take advantage of mutual TLS, you can configure your server to verify Netdata's client certificate. In order to achieve this, the Netdata client sending the notification supports mutual TLS (mTLS) to identify itself with a client certificate that your server can validate.
+
+The steps to perform this validation are as follows:
+
+- Store Netdata CA certificate on a file in your disk. The content of this file should be:
+
+<details>
+  <summary>Netdata CA certificate</summary>
```

Comment:
Maybe this should have a hint that you can expand this section, e.g. by adding ` (click to expand)`. Additionally, it would be nice if we could just download the certificate as a file.